backend/scrapers/classes/parsers/ellucianSectionParser.js

```diff
@@ -175,9 +175,30 @@ class EllucianSectionParser extends ellucianBaseParser.EllucianBaseParser {
       retVal.maxCredits = creditsParsed.maxCredits;
       retVal.minCredits = creditsParsed.minCredits;
     } else {
-      macros.log('warning, nothing matchied credits', url, text);
+      macros.log('warning, nothing matched credits', url, text);
     }
 
+    // grabs optional feees
+    const table = $('body > div.pagebodydiv > table > tr > td > table');
+
+    // for all tables... if table includes word "amount", parse table, get fees/description
+    for (let i = 0; i < table.length; i++) {
+      if ($(table[i]).attr().summary.includes('amount')) {
+	
+        const parsedTable = this.parseTable(table[i]);
+
+	if (parsedTable.rowCount > 0) {
+          retVal.feeDescription = parsedTable.tableData.description[0];
+          retVal.feeAmount = parseFloat(parsedTable.tableData.amount[0]);
+
+	  if (parsedTable.rowCount > 1) {
+	    macros.log('warning... multiple fees?', url);
+	  } else {
+	    macros.log('warning, fees table exists, but fees didn\'t. Nothing parsed.', url);
```

Comment:
Looks like this macros.log should be else'ing the if statement on line 190 and not the one on 194? like move it outside one

Reply:
I'll do it when I wake up. this was the issue from earlier but clearly I out emacsed myself lol